### PR TITLE
chore(gear,vara): Update weights

### PR DESCRIPTION
**Pull request by gshep**
Benchmark 79 - https://github.com/gear-tech/gear/actions/runs/5519781649

@gear-tech/dev

### CODE REVIEW

**Review by ukint-vs — APPROVED** [review]

**Review by shamilsan — APPROVED** [review]

**Comment by breathx** [comment]
Need to adjust tests that are sensitive for weights and :/

File: pallets/gear/src/mock.rs [hunk 1]
```diff
@@ -53,7 +53,7 @@ pub(crate) const USER_3: AccountId = 3;
 pub(crate) const LOW_BALANCE_USER: AccountId = 4;
 pub(crate) const BLOCK_AUTHOR: AccountId = 255;
 const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
-const MAX_BLOCK: u64 = 100_000_000_000;
+const MAX_BLOCK: u64 = 200_000_000_000;
```
Comment (breathx) [inline pallets/gear/src/mock.rs:56]: its 250_000_000_000 for 1s

File: pallets/gear/src/tests.rs [hunk 2]
```diff
@@ -11682,7 +11682,7 @@ fn async_recursion() {
 
     new_test_ext().execute_with(|| {
         let demo = upload();
-        let arg = 100i32;
+        let arg = 40i32;
```
Comment (breathx) [inline pallets/gear/src/tests.rs:11685]: revert as well please

**Review by breathx — APPROVED** [review]

File: pallets/gear/src/tests.rs (hunk 2 above)
Comment (gshep) [reply pallets/gear/src/tests.rs:11685]: then it doesn't fit the block gas

File: pallets/gear/src/mock.rs [hunk 3]
```diff
@@ -210,7 +210,7 @@ impl pallet_gear::Config for Test {
     type Randomness = TestRandomness<Self>;
     type Currency = Balances;
     type GasPrice = GasConverter;
-    type WeightInfo = ();
+    type WeightInfo = pallet_gear::weights::SubstrateWeight<Self>;
```
[the remark this answers is not in the document]
Comment (gshep) [reply pallets/gear/src/mock.rs:213]: Nope, read/write db weights are used from runtime in this case

File: runtime/gear/src/weights/pallet_gear.rs [hunk 4]
```diff
@@ -456,10 +456,20 @@ impl<T: frame_system::Config> pallet_gear::WeightInfo for SubstrateWeight<T> {
...
-        Weight::from_parts(357_675_053, 0)
-            // Standard Error: 1_207_849
-            .saturating_add(Weight::from_parts(437_414_043, 0).saturating_mul(r.into()))
+        // Minimum execution time: 88_275_000 picoseconds.
+        Weight::from_parts(89_733_000, 0)
+            // Standard Error: 3_987_023
+            .saturating_add(Weight::from_parts(669_872_725, 0).saturating_mul(r.into()))
+    }
+    /// The range of component `p` is `[1, 512]`.
+    fn alloc_per_page(p: u32, ) -> Weight {
```
[the remark this answers is not in the document]
Comment (gshep) [reply runtime/gear/src/weights/pallet_gear.rs]: To tell the truth yes, it is. However the new benchmark process will take about 6 hours (

File: pallets/gear/src/mock.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (breathx) [reply pallets/gear/src/mock.rs:213]: > Nope, read/write db weights are used from runtime in this case

ahh, this stuff

**Comment by breathx** [comment]
@gshep feel free to merge than

**Pull request merged**